packages/web/src/components/gcds-input/gcds-input.tsx

```diff
@@ -170,15 +235,11 @@ export class GcdsInput {
    */
   @Watch('aria-invalid')
   ariaInvalidWatcher() {
-    this.inheritedAttributes = inheritAttributes(this.el, this.shadowElement, [
-      'placeholder',
-    ]);
+    this.inheritedAttributes = inheritAttributes(this.el, this.shadowElement);
```

Comment:
Just noticed that `placeholder` is here, and I tested it on the current package and we support it

Reply:
The `inheritedAttributes` hasn't really been something we have talked about or documented. We included `placeholder` in the inherited attributes to encourage some adoption from other teams but that didn't happen. I think it is okay if we were to remove it's support fully for now.